### PR TITLE
Update Lutris to 0.5.8

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build

File: patches/Lutris/metainfo-license.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- a/net.lutris.Lutris.metainfo.xml	2020-11-15 01:09:28.542949609 +0100
++++ b/net.lutris.Lutris.metainfo.xml	2020-11-15 01:09:42.637072978 +0100
+@@ -3,7 +3,7 @@
+ <component type="desktop">
+   <id>net.lutris.Lutris</id>
+   <project_license>GPL-3.0-or-later</project_license>
+-  <metadata_license>GPL-3.0-or-later</metadata_license>
++  <metadata_license>CC0-1.0</metadata_license>
```
Comment (gasinvein) [inline patches/Lutris/metainfo-license.patch:8]: I don't think you can just change license of an upstream file, especially if it's GPL.

Comment (A6GibKm) [reply patches/Lutris/metainfo-license.patch:8]: Yes, I contacted Lutris team for this precise reason, for the moment I just want to see if it builds... I need to fix the patch

**Comment by FakeShemp** [comment]
In my limited testing this seem to work fine :+1:

**Comment by A6GibKm** [comment]
Thank you for the feedback, it seems all it is needed is for me to send a PR upstream and talk with debian people.

**Comment by A6GibKm** [comment]
Upstream issue: https://github.com/lutris/lutris/issues/3277

**Comment by A6GibKm** [comment]
Can confirm, Overwatch launches just fine. Merging as soon as upstream patch is accepted. I am sorry about the delay but *stuff* happened.

**Pull request merged**